t/full-stack.t

```diff
@@ -113,15 +145,19 @@ my $job_page_url = $driver->get_current_url();
 like(status_text, qr/State: scheduled/, 'test 1 is scheduled');
 ok javascript_console_has_no_warnings_or_errors, 'no javascript warnings or errors after test 1 was scheduled';
 
-sub start_worker_and_schedule {
-    $worker = start_worker(get_connect_args());
-    ok($worker, "Worker started as $worker");
-    schedule_one_job;
+sub assign_jobs ($worker_class = undef) {
+    check_scheduled_job_and_wait_for_free_worker $worker_class // 'qemu_i386';
```

Comment:
Since you're using the signature here... would `$worker_class = 'qemu_i386'` work here?

Reply:
No, this way I'd needed to hard-code the string `'qemu_i386'` in two places (because both functions should have that default value).

Reply:
To be clear, what I meant is to use the signature so the `//` in the next line isn't needed:

```suggestion
sub assign_jobs ($worker_class = 'qemu_i386') {
    check_scheduled_job_and_wait_for_free_worker $worker_class;
```

Reply:
Yes, but then I need to do this in both of the functions and would duplicate the string.

Reply:
That isn't what I suggested. But feel free to leave it as-is. It's not a big difference.